src/container/oid_iv.c

```diff
@@ -133,10 +134,10 @@ oid_iv_ent_update(struct ds_iv_entry *ns_entry, struct ds_iv_key *iv_key,
 	entry = ns_entry->iv_value.sg_iovs[0].iov_buf;
 	rc = ABT_mutex_trylock(entry->lock);
 	/** For retry requests, from _iv_op(), the lock may not be released in some cases. */
```

Comment:
is it guranteed that priv going to be the same for retry requests? i am not sure what daos stores in it or how it manages it across requests

Reply:
yes i believe so. the priv is on the on-alloc request. so it will not change on retry afaik.